WooCommerce/src/test/kotlin/com/woocommerce/android/tracker/SendTelemetryTest.kt

```diff
@@ -156,8 +173,10 @@ class SendTelemetryTest : BaseUnitTest() {
         assertThat(results).containsExactly(SENT, NOT_SENT, SENT)
     }
 
-    companion object {
+    private companion object {
         val site = SiteModel().apply { id = 123 }
         val siteB = SiteModel().apply { id = 321 }
+        val APP_INSTALLATION_DATE = Date(1692011562123) // 14 2023 11:12:42 UTC
```

Comment:
⛏️ extreme minor 😅 :
```suggestion
        val APP_INSTALLATION_DATE = Date(1692011562123) // 14 Aug 2023 11:12:42 UTC
```

Reply:
Good eye!! I accidentally removed the month at some point 😅